README.md

```diff
@@ -22,6 +22,8 @@ Feel free to explore the code and assets in this repository, and don't hesitate
 
 ## 💟 Preview of the Webpage
 
+![404-page](https://github.com/sidhartha2002/crane-404-error-page/assets/73163725/83769a20-6c10-415c-a70e-7ff2d253fb6a)

```

Comment:
Great job! You have done more than I expected. I have a request about the link because it leads to external resources, but there is a much easier way to follow.

This works in Markdown+HTML5 and is easy to understand when looking at the source code.

The code: `[<img src="assets/example.png">] (https://portfolio.darekduda.pl/crane-404-error-page/)`

where `img/example.png` should be replaced by the relative path to your image (it is done by simply adding a new folder and copying this image), and `https://portfolio.darekduda.pl/crane-404-error-page/` should be replaced by the link you want to visit when the image is clicked.

I updated the repo to show you the example. Check this out. It is in the picture paragraph.

Reply:
Ok bro that will be great. I'm doing the specified changes.